terraform/cloudfront.tf

```diff
@@ -0,0 +1,42 @@
+resource "aws_cloudfront_distribution" "s3_distribution" {
+  tags        = module.beacons_label.tags
+  origin {
+    domain_name = aws_s3_bucket.backoffice-static.bucket_regional_domain_name
+    origin_id   = var.name
+  }
+
+  enabled             = true
+  default_root_object = "index.html"
+
+  default_cache_behavior {
+    allowed_methods  = ["DELETE", "GET", "HEAD", "OPTIONS", "PATCH", "POST", "PUT"]
+    cached_methods   = ["GET", "HEAD"]
+    target_origin_id = var.name
+
+    forwarded_values {
+      query_string = true
+
+      cookies {
+        forward = "none"
+      }
+    }
+
+    viewer_protocol_policy = "redirect-to-https"
+    min_ttl                = 0
+    default_ttl            = 3600
+    max_ttl                = 86400
+  }
+
+  price_class = "PriceClass_100"
+
+  restrictions {
+    geo_restriction {
+      restriction_type = "whitelist"
+      locations        = ["GB"]
+    }
+  }
```

Comment:
Set to only serve to locations in GB for a bit of extra security.  What do we think?

Reply:
Probably not needed, especially if we are having auth in place as well.

One for @paulgutman to comment on

Reply:
Agreed, I think  basic auth is fine